petasus/mzml.py

```diff
@@ -8,7 +8,8 @@
 def read(mzml_file: str) -> dict[tuple[np.ndarray, np.ndarray]]:
     """Parse the mzML file
 
-    This parser is much faster than Pyteomics, but doesn't do any error checking.
+    This parser is much faster than Pyteomics, but doesn't do any error
+    checking.
```

Comment:
Request: Can you also add a mention that it handles less types of mzml's ... it assumes gzip compressed arrays every time and assumes its all 64 bit.